tests/integration/dual-channel-replication.tcl

```diff
@@ -1197,3 +1197,69 @@ start_server {tags {"dual-channel-replication external:skip"}} {
         stop_write_load $load_handle
     }
 }
+
+
+start_server {tags {"dual-channel-replication external:skip"}} {
+    set primary [srv 0 client]
+    set primary_host [srv 0 host]
+    set primary_port [srv 0 port]
+    set loglines [count_log_lines 0]
```

Comment:
You set this value later, so it has no impact here.

Reply:
I use this between the two sets for 
```
wait_for_log_messages 0 {"*Loading RDB produced by Valkey version*"} $loglines 1000 10
```

Reply:
Yes, but you set it again on line 1228